### ISSUE TITLE
Aria2 external downloader

### ISSUE THREAD

**Issue by shuvashish76** [SkyD666/PodAura#86]
Similar to [Seal](https://github.com/JunkFood02/Seal) support aria2 (supports _BitTorrent_) as external downloader.
![Screenshot_2024-08-09_18-26-05_1](https://github.com/user-attachments/assets/87873d8f-c534-424d-bea8-fe947916d23c)

https://github.com/aria2/aria2

**Comment by SkyD666** [comment]
The aria2 in Seal seems to be built-in, do you mean you also want PodAura to have aria2 built-in?

**Comment by shuvashish76** [comment]
Yes. If it increases the size significantly then probably not worth for it.

**Comment by SkyD666** [comment]
> Yes. If it increases the size significantly then probably not worth for it.

Approximately 7MB increase

**Comment by shuvashish76** [comment]
Then it's up-to you to decide if it is worth for the users.

**Comment by SkyD666** [comment]
Okay, I won't plan to do that for now.
